src/pages/Home.test.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import App from '../App'
+import Home from './Home'
+
+Enzyme.configure({adapter: new Adapter()})
+
+describe("Check For Home Component...",() => {
+  it("As a developer, I have test coverage on my Home component.", () => {
+    const renderedApp = shallow(<App/>)
```

Comment:
The Home test should be testing the Home Component not the App component. Feel free to move this test into the App.test.js.

Reply:
We made changes to the Home.test.js to actually reference the home component.